manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -6370,6 +6370,8 @@ spec:
                                 items:
                                   type: string
                                 type: array
+                              ignorePathCase:
```

Comment:
Do you have a design document for this feature?

Reply:
No, just saw the issue when I first joined and implemented it as I was ramping up.  This was just sent out as an example of how it might be done.  If it's satisfactory then I will wrap it up with whatever needs to be done.

Reply:
Thanks, Brian. Can you draft a design doc for review?